src/services/game/toggleModeRulesVisibility.js

```diff
@@ -0,0 +1,8 @@
+import changeButtonText from './changeButtonText';
+
+export default function toggleGameRulesVisibility() {
```

Comment:
remove default export

Reply:
When i remove default export and then import function using { toggleGameRulesVisibility } then this import is undefined and I can't find why. On default export and import without {} it's work